Assets/World/Global.gd

```diff
@@ -341,49 +345,59 @@ const COLOR_MATERIAL = [
 
 const MESSAGE_SCENE = preload("res://Assets/UI/Scenes/Message.tscn")
 
-const CONFIG_FILE = "user://unknown_horizon.ini"
+#const WINDOW_MODES = [
+#	WindowMode.WINDOWED,
+#	WindowMode.FULLSCREEN
+#]
 
-#warning-ignore-all:unused_class_variable
+const WINDOW_MODES = {
+	WindowMode.WINDOWED: "Windowed",
+	WindowMode.FULLSCREEN: "Fullscreen"
+}
+
+# Screen resolution choices
+const SCREEN_RESOLUTIONS = [
+	"800x600",
+	"1024x768",
+	"1280x1024",
+	"1280x720",
+	"1280x800",
+	"1360x768",
+	"1366x768",
+	"1440x900",
+	"1600x900",
+	"1680x1050",
+	"1920x1200",
+	"1920x1080",
+	"2560x1080",
+	"2560x1440",
+	"3440x1440",
+	"3840x2160",
+]
```

Comment:
Why not put this in `Assets/UI/Scripts/OptionsUI.gd`? It's only used for displaying the options.